app/routes/bundesIdent/erfolgreich.tsx

```diff
@@ -22,26 +21,8 @@ export const loader: LoaderFunction = async ({ request }) => {
   if (!dbUser.identified) {
     return redirect("/identifikation");
   }
-
-  const session = await getSession(request.headers.get("Cookie"));
-  const hasRatingPageShown = Boolean(session.get("hasRatingPageShown"));
-  session.set("hasRatingPageShown", hasRatingPageShown);
-
-  return {
-    hasRatingPageShown,
-    isMobile: isMobileUserAgent(request),
-  };
```

Comment:
this caused an issue that the page won't render. A loader function (from error message) should return object, if none is returned, an empty object should be returned.

```js
Error: You defined a loader for route "routes/bundesIdent/erfolgreich" but didn't return anything from your `loader` function. Please return a value or `null` or empty object, since we use the empty object in the project, I put the empty object here
    at Object.callRouteLoaderRR (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/server-runtime/dist/data.js:47:11)
    at callLoaderOrAction (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/router/router.ts:3540:16)
    at async Promise.all (index 2)
    at loadRouteData (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/router/router.ts:2974:19)
    at queryImpl (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/router/router.ts:2753:20)
    at Object.query (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/router/router.ts:2629:18)
    at handleDocumentRequestRR (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/server-runtime/dist/server.js:146:15)
    at requestHandler (/Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/server-runtime/dist/server.js:49:18)
    at /Users/pramgurusinga/Workspace/DigitalService/grundsteuer/node_modules/@remix-run/express/dist/server.js:39:22
```